pocs/base.py

```diff
@@ -10,6 +10,16 @@
 _config = None
 
 
+def reset_config_for_test():
```

Comment:
What about just making this an option that is passed to PanBase, like `ignore_local_config`? I can imagine there might be a situation where someone needs to reset the config for whatever reason. Or just change this function to `reset_config`.

Reply:
There are many instances of PanBase in a running POCS system, and it doesn't make sense (to me) for them to have different global configs.

I suggested elsewhere that we might have a PanConfig object that gets passed around, and represents whatever config has been loaded by the creator of POCS and Observatory. This fix is designed to allow us to make a decision about that latter, yet still get closer to hermetic tests now.

So, I'll rename to reset_global_config, with a comment that it is intended for use in tests, but might have other uses.